content/docs/setup/kubernetes/quick-start/index.md

```diff
@@ -118,17 +106,6 @@ Follow our instructions on how to
 
```

Comment:
Tricky, I know, but if you remove the output you need to change the text in the step too.
1.  Ensure the corresponding Kubernetes pods are deployed and the status of the following containers
     is shown as `Running`: `istio-pilot-*`, `istio-ingressgateway-*`,
     `istio-egressgateway-*`, `istio-policy-*`, `istio-telemetry-*`,
     `istio-citadel-*`, `prometheus-*`, `istio-galley-*`, and, optionally,
     `istio-sidecar-injector-*`.

Reply:
Fixed :metal: